reddit_liveupdate/templates/liveupdatediscussionslisting.html

```diff
@@ -0,0 +1,91 @@
+<%!
+  from r2.lib.strings import Score
+  from r2.lib.filters import safemarkdown
+%>
+
+<h1>${c.liveupdate_event.title}</h1>
+
+<style>
+  p {
+    font-size: 13px;
+  }
+
+  #discussions {
+    margin-top: 1em;
+    font-size: 13px;
+  }
+
+  #discussions th {
+    font-size: 15px;
+  }
+
+  #discussions th,
+  #discussions td {
+    padding: .5em 2em 0 0;
+  }
+
+  #discussions .pretty-button {
+    margin-bottom: 0;
+  }
+</style>
+
+% if not thing.things:
+${unsafe(safemarkdown(_("no discussions yet. [start one]({url})").format(url=thing.submit_url), wrap=False))}
```

Comment:
You can just use [`_mdf()`](https://github.com/reddit/reddit-public/blob/ccd76359984d26f19dc64b62024d6dbc5cbdd4c6/r2/r2/templates/confirmawardclaim.html#L32) here to get an internationalized markdown template

Reply:
Will do. I'll update the older use case this was a copy-paste of at the same time.

Reply:
Done